CODEOWNERS

```diff
@@ -13,18 +13,54 @@
 /cmd/common/ @flyth
 /cmd/ig/ @blanquicet
 /Dockerfiles/ @eiffel-fl @mqasimsarfraz
+
+# docs
 /docs/ @mauriciovasquezbernal
+/docs/gadget-devel/ @mauriciovasquezbernal
 /docs/images/architecture/ @eiffel-fl
+
 /examples/ @mauriciovasquezbernal
 /examples/container-collection/ @alban
 /examples/container-hook/ @alban
 /examples/kube-container-collection/ @alban
 /examples/runc-hook/ @alban
 /gadget-container/hooks/ @alban
-/gadgets/ @mauriciovasquezbernal
+
+# gadgets
+/gadgets/audit_seccomp/ @alban
+/gadgets/ci/ @mauriciovasquezbernal
+/gadgets/deadlock/ @alban
+/gadgets/fsnotify/ @alban
+/gadgets/profile_blockio/
```

Comment:
Someone is missing here (if this wasn't intended)

Reply:
Good catch, thanks!